examples/static/index.html

```diff
@@ -14,7 +14,7 @@
     <li><a href="microphone-word-confidence.html">Transcribe from Microphone, with Word Confidence</a></li>
     <li><a href="microphone-streaming-text-to-console.html">Transcribe from Microphone, send text to console</a></li>
     <li><a href="microphone-streaming-object-to-console.html">Transcribe from Microphone, send JSON to console (includes text and metadata)</a></li>
-    <li><a href="microphone-streaming.html">Transcribe from Microphone, Streaming</a></li>
```

Comment:
Oh, I just realized that there were two entries for micrphone-streaming.html in the list. This makes sense now.